### PR TITLE
ProgressBar: Remove aria-readonly attribute

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.tests.js [hunk 1]
```diff
@@ -201,6 +201,14 @@ QUnit.module('states', {
 });
 
 QUnit.module('aria accessibility', () => {
+    QUnit.test('aria-readonly is removed in initialization', function(assert) {
+        const $element = $('#progressbar').dxProgressBar({
+            readOnly: true,
+        });
+
+        assert.equal($element.attr('aria-readonly'), undefined, 'aria-readonly is removed');
+    });
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.tests.js:239]: does it work after value is changed?

File: packages/devextreme/js/__internal/ui/m_progress_bar.ts [hunk 2]
```diff
@@ -184,6 +185,10 @@ const ProgressBar = TrackBar.inherit({
       case 'statusPosition':
         this._toggleStatus(this.option('showStatus'));
         break;
+      case 'readOnly':
+        this.callBase(args);
+        this.setAria('readonly', undefined);
```
Comment (ksercs) [inline packages/devextreme/js/__internal/ui/m_progress_bar.ts]: cannot we refactor it to not manually update element's attr in 2 places?
For example override some of method implemented on editor's level?
Now attribute is set somewhere in `callBase` and then u restore it here. Cannot we just not set it?

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.tests.js [hunk 3]
```diff
@@ -201,6 +201,43 @@ QUnit.module('states', {
 });
 
 QUnit.module('aria accessibility', () => {
+    QUnit.test('aria-readonly is removed in initialization', function(assert) {
+        const $element = $('#progressbar').dxProgressBar({
+            readOnly: true,
+        });
+
+        assert.equal($element.attr('aria-readonly'), undefined, 'aria-readonly is removed');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.tests.js]: fix everywhere please
```suggestion
        assert.strictEqual($element.attr('aria-readonly'), undefined, 'aria-readonly is not set');
```

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.tests.js]: [ab1c9f1](https://github.com/DevExpress/DevExtreme/pull/27984/commits/ab1c9f1c45526c7aeaee0d7864c1d5a772f50a40)

File: packages/devextreme/js/__internal/ui/m_progress_bar.ts (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/__internal/ui/m_progress_bar.ts]: [ab1c9f1](https://github.com/DevExpress/DevExtreme/pull/27984/commits/ab1c9f1c45526c7aeaee0d7864c1d5a772f50a40)

**Review by marker-dao — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**